sat/cli/showrev/system.py

```diff
@@ -82,7 +82,12 @@ def get_site_data(sitefile):
 
     try:
         with open(sitefile, 'r') as f:
-            data.update(yaml.safe_load(f.read()))
+            site_info = yaml.safe_load(f)
+            if site_info is None:
+                LOGGER.warning('Site info file "%s" is empty; run `sat setrev` to populate.',
```

Comment:
The other cases (file does not exist, file cannot be opened, file is invalid YAML) log an error and return `default`. Should this be error-level as well?